docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -1097,6 +1097,64 @@ delegate :foo, to: :bar, allow_blank: true
 delegate :foo, to: :bar, allow_nil: true
 ----
 
+== Rails/DeleteBy
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| Yes
```

Comment:
There's been a few cases recently where a cop has been released as `safe` then marked as `unsafe` once issues were discovered. Perhaps it would be better to initially launch as `unsafe` then change to `safe` after some time?

Reply:
Ah, it may be falsely detected by AR-like modules like Mongoid. This setting updates by unsafe by default.

Reply:
Isn't replacing `.find_by(readable: readable)&.destroy` by `.destroy_by(readable: readable)` extremely unsafe?

The latter form deletes potentially more records than the original (limited to at most 1)

Reply:
I think the replacement is safe-autocorrection, but I was worried about false positive from the receiver type.
However, I think that the possibility of false positives in this method chain is low, so I think it is possible to start from `Safe: true`.